core-overlay.html

```diff
@@ -111,666 +124,373 @@
   .core-overlay-backdrop.core-opened {
     opacity: 0.6;
   }
+
 </style>
 
-<polymer-element name="core-overlay">
+<dom-module id="core-overlay">
+
+  <style>
+
+    :host {
+      position: fixed;
+      background: #fff;
+    }
+
+  </style>
+
+</dom-module>
+
 <script>
-(function() {
 
-  Polymer(Polymer.mixin({
+  (function() {
+
+    var manager = Polymer.Core.Overlay.Manager;
+
+    Polymer({
+
+      is: 'core-overlay',
+
+      mixins: [
+        Polymer.Core.Overlay.Positioner,
+        Polymer.Core.EventUtils
+      ],
+
+      properties: {
+
+        /**
+         * A `core-overlay`'s size is guaranteed to be
+         * constrained to the window size. To achieve this, the sizingElement
+         * is sized with a max-height/width. By default this element is the
+         * target element, but it can be specifically set to a specific element
+         * inside the target if that is more appropriate. This is useful, for
+         * example, when a region inside the overlay should scroll if needed.
```

Comment:
After reading this, I'm still not sure what will happen if I change `sizingTarget`. Does it mean that the element I set it to will govern the dimensions of the overlay?